### PR TITLE
Add additional paramater for discoverability endpoint

### PR DESCRIPTION
**Pull request by henryc132**

### CODE REVIEW

File: Spec/API.md [hunk 1]
```diff
@@ -355,9 +355,12 @@ Response:
 ### Search
 Request:
 ```
-GET /api/v1/discover/search?query=<query>[&page=<page>]
+GET /api/v1/discover/search?query=<query>&tags=<tags>[&page=<page>]
 ```
```
Comment (henryc132) [inline Spec/API.md:359]: Separated the title search from the tag search

File: Spec/API.md [hunk 2]
```diff
@@ -339,7 +339,7 @@ Request:
...
 GET /api/v1/discover/new[?page=<page>]
 ```
-Where `page` is the page offset into the result set (0 by default).
+Where `page` is the page offset into the result set (1 by default).
```
Comment (henryc132) [inline Spec/API.md:343]: Changed to match code, default page is 1. Thought it made more sense to start at 1 in this case.

**Comment by ClaytonPassmore** [comment]
@henryc132 is the `page` parameter optional or required? If it's required, you should drop the square brackets around it in the request. Other than that, looks good

**Review by ClaytonPassmore — APPROVED** [review]

**Comment by henryc132** [comment]
Its optional, you just get return first page results if you don't specify it

**Pull request merged**